tools/pipelines/test-perf-benchmarks.yml

```diff
@@ -537,16 +545,81 @@ stages:
+              echo "FLUID_TEST_LOGGER_PKG_PATH= ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger"
+              echo "FLUID_BUILD_ID ${{ variables.artifactBuildId }}"
+
+              # Run execution time tests
+              echo "FLUID_ENDPOINTNAME = odsp"
+              npm run test:benchmark:report:odsp;
+              executionTimeTestsExitCodeOdsp=$?;
+
+              # Run memory tests
+              echo "FLUID_ENDPOINTNAME = odsp"
+              npm run test:memory-profiling:report:odsp;
+              memoryTestsExitCodeOdsp=$?;
+
+              if [[ $executionTimeTestsExitCodeOdsp -ne 0 ]]; then
+                echo "##vso[task.logissue type=error]Exit code for runtime tests execution Odsp= $executionTimeTestsExitCodeOdsp"
+              fi
+
+              if [[ $memoryTestsExitCodeOdsp -ne 0 ]]; then
+                echo "##vso[task.logissue type=error]Exit code for memory tests execution Odsp = $memoryTestsExitCodeOdsp"
+              fi
+
+              if [[ $executionTimeTestsExitCodeOdsp -ne 0 ]] ||  [[ $memoryTestsExitCodeOdsp -ne 0 ]]; then
+                exit 1;
+              fi
+          env:
+            login__microsoft__clientId: $(login-microsoft-clientId)
+            login__microsoft__secret: $(login-microsoft-secret)
+            login__odsp__test__tenants: $(automation-stress-login-odsp-test-tenants)
+            FLUID_TEST_LOGGER_PKG_PATH: ${{ variables.testWorkspace }}/node_modules/@ff-internal/aria-logger # Contains getTestLogger impl to inject
+            FLUID_BUILD_ID: ${{ variables.artifactBuildId }}
+            FLUID_ENDPOINTNAME: odsp
+
+        - task: Bash@3
+          displayName: Write measurements to Aria/Kusto - execution time odsp
+          condition: succeededOrFailed()
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              echo "Writing the following performance tests results to Aria/Kusto - ODSP"
+              ls -la ${{ variables.executionTimeTestOutputFolder }};
+              node --require @ff-internal/aria-logger bin/run --handlerModule $(toolAbsolutePath)/dist/handlers/executionTimeTestHandler.js --dir ${{ variables.executionTimeTestOutputFolder }};
+          env:
+            FLUID_ENDPOINTNAME: "odsp"
+
+        - task: Bash@3
+          displayName: Write measurements to Aria/Kusto - memory usage odsp
+          condition: succeededOrFailed()
+          inputs:
+            targetType: 'inline'
+            workingDirectory: $(toolAbsolutePath)
+            script: |
+              echo "Writing the following performance tests results to Aria/Kusto - ODSP"
+              ls -la ${{ variables.memoryUsageTestOutputFolder }};
+              node --require @ff-internal/aria-logger bin/run --handlerModule $(toolAbsolutePath)/dist/handlers/memoryUsageTestHandler.js --dir ${{ variables.memoryUsageTestOutputFolder }};
+          env:
+            FLUID_ENDPOINTNAME: "odsp"
```

Comment:
Slight concern about these steps (and implicitly, the one above that runs the tests) using the same output directories (`variables.executionTimeTestOutputFolder`, `variables.memoryUsageTestOutputFolder`) than the corresponding steps using the local endpoint. The files might be getting overwritten right now, but if there are differences in which tests run against local vs against odsp, we could end up grabbing extra files here, so I think we should add a cleanup step in between steps for local and odsp endpoints, or have the odsp ones use separate output folders (which I think might be possible by changing the invocation from `npm run test:memory-profiling:report:odsp` to something like `npm run test:memory-profiling:report:odsp -- --outputDirectory <some-path>` or similar to that; don't quote me on the exact syntax or flag names).

Reply:
Good point ! Removing intermediate files.